docs/source/support/release-notes.rst

```diff
@@ -14,6 +14,12 @@ DAML Integration Kit
 
 - Added new CLI flag ``--all-tests`` to the :doc:`Ledger API Test Tool </tools/ledger-api-test-tool/index>` to run all default and optional tests.
 
+Ledger
```

Comment:
@cbaatz / @cbaatz-da: should we call this `Ledger` or `Sandbox`?

Reply:
Technically #1312 was for implementing the party management in the `Sandbox`. But the release notes so far do not mention the addition of the new ledger API endpoints (which is the important bit for the end user), so I put it under `Ledger`.

Reply:
Changed to `Sandbox`, following slack discussions.